code/game/objects/items/weapons/tools/misc.dm

```diff
@@ -52,6 +52,36 @@
 	max_upgrades = 1
 	spawn_tags = null
 
+/obj/item/tool/multitool_improvised
+	name= "improvised multitool implant"
+	desc = "A jury-rigged implant, holding cobbled-together tools. For those who are more interested in tool carrying than scared of tetanus."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "multitool_improvised"
+	force = WEAPON_FORCE_WEAK
+	switched_on_force = WEAPON_FORCE_PAINFUL * 0.8
+	worksound = WORKSOUND_HARD_SLASH
```

Comment:
This one doesn't sound quite right, imo. Consider using another, less organic sound.
```suggestion
	worksound = WORKSOUND_DRIVER_TOOL
```
That's what other omnitools use, sounds dope.